src/main/java/dook/commands/Command.java

```diff
@@ -0,0 +1,18 @@
+package dook.commands;
+
+import java.io.IOException;
+
+import dook.DookException;
+import dook.storage.Storage;
+import dook.tasks.TaskList;
+import dook.ui.Ui;
+
+public abstract class Command {
```

Comment:
Could it be a good idea to use Javadoc header comments as it is a non-trivial public class?